transformers/events/pot_join/pot_join_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package pot_join_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestPotJoin(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "PotJoin Suite")
+}
```

Comment:
I was going to say we should `logrus.SetOutput(ioutil.Discard)`, but that actually doesn't seem necessary. I wonder if we can delete this from other places if transformers are just converters and converters aren't logging? 🤔